src/kaleidoscope/plugin/LEDControl.h

```diff
@@ -85,12 +85,12 @@ class LEDControl : public kaleidoscope::Plugin {
     return 0;
   }
 
-  static void setCrgbAt(int8_t led_index, cRGB crgb);
+  static void setCrgbAt(uint8_t led_index, cRGB crgb);
```

Comment:
If you change this anyway, I would go with a `LEDIndex` class instead of `uint8_t`, to make things more versatile.

Reply:
I suppose I could do that. We'd still need an `uint8_t` wrapper though, for compatibility, but could perhaps deprecate that.